src/navigation/reroute.js

```diff
@@ -10,12 +10,26 @@ import { getAppsToUnload, toUnloadPromise } from '../lifecycles/unload.js';
 
 let appChangeUnderway = false, peopleWaitingOnAppChange = [];
 
+const SKIP_APPS_WITH_LOAD_ERRORS = 'SKIP_APPS_WITH_LOAD_ERRORS';
+
 export function triggerAppChange() {
   // Call reroute with no arguments, intentionally
   return reroute()
 }
 
-export function reroute(pendingPromises = [], eventArguments) {
+export function reroute(...args) {
+  let pendingPromises,
+    eventArguments,
+    skipAppsWithLoadErrors = args[0] === SKIP_APPS_WITH_LOAD_ERRORS;
+
+  if (skipAppsWithLoadErrors) {
```

Comment:
Eventually reroute will hit 176 https://github.com/CanopyTax/single-spa/pull/342/files#diff-7fbcab994465f5a2c61f4cb367998227R176 and now that invocation that re-calls reroute passes this flag. If this flag is set, then don't retry loading apps that failed with `LOAD_ERROR`